### ISSUE TITLE
including dockerfiles for MCnet tools?

### ISSUE THREAD

**Issue by lukasheinrich** [hepsw/docks#13]
Hi again,

would you like to build this into a general repository for HEP software tools. I have a couple of Dockerfiles for MCNet tools like Sherpa/Herwig/Madgraph/Rivet and other things like ROOT etc and I'd be happy to contribute.

Cheers,
Lukas

**Comment by sbinet** [comment]
sure, send them in. or, rather, PR-them in :)

**Comment by lukasheinrich** [comment]
Hi,

so I have this repo: 

https://github.com/lukasheinrich/hepsw-docker

would you like to follow a specific scheme on the dockerfiles?

**Comment by sbinet** [comment]
looks good.

what do you mean by "follow[ing] a specific scheme on the dockerfiles" ?

- a standard installation path for the binaries (inside the container) ?
- a standard installation location for the sources ?
- a standard location or naming scheme for the `Dockerfile`s inside the `hepsw/docks` repository ?

**Comment by lukasheinrich** [comment]
yeah possibly any of those. the existing one seem to come with a Dockerfile + a Makefile to do build and push. Also maybe could be a good idea to try to systematically build base layers. E.g I did try to use the slc-dev image for some stuff, but needed to compile gcc4.9 on top. 

we can also just let it be free-form, just wanted to ask before I start putting in PRs

**Comment by sbinet** [comment]
ok. let's start with indeed a `Dockerfile` + `Makefile` to streamline the build+upload process.

as for the layout of the directories holding these `Dockerfiles`, I would just start relatively flat (except if you want to put the mcnet related tools under, say, `mctools` or something.)
we can still `git mv` stuff around.

for the sources being installed inside the container (if any), I would put them under `/opt/hepsw/<some-name>` and install the binaries+headers under `/usr/{include,lib,bin}`
